proof/records.go

```diff
@@ -54,6 +54,12 @@ const (
 	MetaRevealUniverseCommitments    tlv.Type = 7
 	MetaRevealCanonicalUniversesType tlv.Type = 9
 	MetaRevealDelegationKeyType      tlv.Type = 11
+
+	SendFragmentVersionType     tlv.Type = 0
```

Comment:
Arguably could live in the `address` package, as IIUC we're not placing this information into proofs _yet_.

Reply:
Ah yes, makes sense.